packages/core/package.json

```diff
@@ -7,7 +7,6 @@
     "simplebar-react": "^2.1.0"
   },
   "peerDependencies": {
-    "@material-ui/core": "^4.9.3",
```

Comment:
This should be replace with `@mui/material`, no?

Reply:
As per the [docs](https://mui.com/material-ui/migration/migration-v4/#:~:text=%40material%2Dui/core%20%2D%3E%20%40mui/material), I think so.